faster_whisper/vad.py

```diff
@@ -3,47 +3,62 @@
+    are considered as SPEECH. It is better to tune this parameter for each dataset separately,
+    but "lazy" 0.5 is pretty good for most datasets.
+    """
+    min_speech_duration_ms: int = 250
+    """Final speech chunks shorter min_speech_duration_ms are thrown out.
+    """
+    max_speech_duration_s: float = float("inf")
+    """Maximum duration of speech chunks in seconds.
+
+    Chunks longer than max_speech_duration_s will be split at the timestamp of the last silence
+    that lasts more than 100s (if any), to prevent agressive cutting. Otherwise, they will be
+    split aggressively just before max_speech_duration_s.
+    """
+    min_silence_duration_ms: int = 2000
+    """In the end of each speech chunk wait for min_silence_duration_ms before separating it.
+    """
+    window_size_samples: int = 1024
+    """Audio chunks of window_size_samples size are fed to the silero VAD model.
+
+    WARNING! Silero VAD models were trained using 512, 1024, 1536 samples for 16000 sample rate.
+    Values other than these may affect model performance!!
+    """
+    speech_pad_ms: int = 400
+    """Final speech chunks are padded by speech_pad_ms each side.
+    """
 
 
-def get_speech_timestamps(
-    audio: np.ndarray,
-    *,
-    threshold: float = 0.5,
-    min_speech_duration_ms: int = 250,
-    max_speech_duration_s: float = float("inf"),
-    min_silence_duration_ms: int = 2000,
-    window_size_samples: int = 1024,
-    speech_pad_ms: int = 400,
-) -> List[dict]:
+def get_speech_timestamps(audio: np.ndarray, vad_options: VadOptions) -> List[dict]:
     """This method is used for splitting long audios into speech chunks using silero VAD.
 
     Args:
       audio: One dimensional float array.
-      threshold: Speech threshold. Silero VAD outputs speech probabilities for each audio chunk,
-        probabilities ABOVE this value are considered as SPEECH. It is better to tune this
-        parameter for each dataset separately, but "lazy" 0.5 is pretty good for most datasets.
-      min_speech_duration_ms: Final speech chunks shorter min_speech_duration_ms are thrown out.
-      max_speech_duration_s: Maximum duration of speech chunks in seconds. Chunks longer
-        than max_speech_duration_s will be split at the timestamp of the last silence that
-        lasts more than 100s (if any), to prevent agressive cutting. Otherwise, they will be
-        split aggressively just before max_speech_duration_s.
-      min_silence_duration_ms: In the end of each speech chunk wait for min_silence_duration_ms
-        before separating it
-      window_size_samples: Audio chunks of window_size_samples size are fed to the silero VAD model.
-        WARNING! Silero VAD models were trained using 512, 1024, 1536 samples for 16000 sample rate.
-        Values other than these may affect model perfomance!!
-      speech_pad_ms: Final speech chunks are padded by speech_pad_ms each side
-
+      vad_options: Options for VAD processing
```

Comment:
Just to keep the newline here:

```suggestion
      vad_options: Options for VAD processing.

```

Reply:
Fixed.